pages/linux/gpclient.md

```diff
@@ -0,0 +1,33 @@
+# gpclient
+
+> Client for using GlobalProtect VPN on Linux via OpenConnect.
+> `gpclient` allows you to connect, disconnect, and manage VPN connections using a command-line interface or a graphical user interface (GUI).
```

Comment:
```suggestion
> Connect to a GlobalProtect VPN on Linux via OpenConnect.
```
Keep the description as brief as possible. The commands will document themselves. If possible, use imperative mood.